TASVideos.ForumEngine/BbParser.cs

```diff
@@ -166,12 +171,12 @@ public enum SelfNestingAllowed
 
 		// list related stuff
 		{ "list", new() { IsBlock = true } }, // OLs have a param with value ??
-		{ "*", new() { SelfNesting = TagInfo.SelfNestingAllowed.NoImmediate, IsBlock = true } },
+		{ "*", new() { SelfNesting = TagInfo.SelfNestingAllowed.NoImmediate, IsBlock = true, RequiredParent = "list" } },
 
 		// tables
 		{ "table", new() { SelfNesting = TagInfo.SelfNestingAllowed.No, IsBlock = true } },
-		{ "tr", new() { SelfNesting = TagInfo.SelfNestingAllowed.No, IsBlock = true } },
-		{ "td", new() { SelfNesting = TagInfo.SelfNestingAllowed.No, IsBlock = true } },
+		{ "tr", new() { SelfNesting = TagInfo.SelfNestingAllowed.No, IsBlock = true, RequiredParent = "table" } },
```

Comment:
Yeah, tables were implicitly supported without the `table` tag, so I feel making it required will break existing tables :(